adapters/dynamoDB_adapters/src/main/java/org/atomhopper/dynamodb/model/PersistedEntry.java

```diff
@@ -0,0 +1,21 @@
+package org.atomhopper.dynamodb.model;
+import com.amazonaws.services.dynamodbv2.datamodeling.*;
+import lombok.Data;
+import java.util.*;
+@Data
+@DynamoDBTable(tableName = "entries")
+public class PersistedEntry {
+
+    @DynamoDBHashKey(attributeName = "entryId")
+    private String entryId;
+    @DynamoDBIndexRangeKey(attributeName = "feed",
+            localSecondaryIndexName = "entryId-feed-index")
+    @DynamoDBIndexHashKey(globalSecondaryIndexName="global-feed-index", attributeName = "feed")
```

Comment:
Again, GSI can be appended to the name to specify the type of the index.
feed-GSI or feed-global-index

Reply:
ok..done